deploy/complete/kubernetes/mushop.yaml

```diff
@@ -1216,7 +1216,8 @@ spec:
         
       containers:
         - name: orders
-          image: "iad.ocir.io/oracle/ateam/mushop-orders:2.2.2"
+          #image: "iad.ocir.io/oracle/ateam/mushop-orders:2.2.2"
+          image: "phx.ocir.io/ociateam/mushop-orders:4.0"
```

Comment:
We cannot use non-official tenancies, and gets worse if you change the main manifest with that. Our CI automatically pushes the Microservices image to the official OCIR, if you think that is not using the latest, change just the version.

Reply:
yes agree, this is just a placeholder for now.

Reply:
a placeholder on the main mushop values (deploy/complete/kubernetes/mushop.yaml). Need to change before merge.